catroid/src/test/java/org/catrobat/catroid/test/content/actions/GoToRandomPositionActionTest.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2022 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.test.content.actions
+
+import junit.framework.Assert
+import org.catrobat.catroid.common.BrickValues
+import org.catrobat.catroid.common.ScreenValues
+import org.catrobat.catroid.content.ActionFactory
+import org.catrobat.catroid.content.Sprite
+import org.catrobat.catroid.content.actions.GoToRandomPositionAction
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.ExpectedException
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+class GoToRandomPositionActionTest {
+    @JvmField
+    @Rule
+    val exception: ExpectedException = ExpectedException.none()
+
+    private var sprite: Sprite? = null
```

Comment:
I would prefer, if all the three variables would be initialized with the `lateinit` key word as follows:
```suggestion
    private lateinit var sprite: Sprite
```
Gives the code a cleaner look, since we you can then remove all the `!!` operators from the method calls.